src/lib/signature_lib/test/test_private_key.ml

```diff
@@ -0,0 +1,312 @@
+open Core_kernel
+open Signature_lib
+open Base_quickcheck
+
+let modulus_string =
+  "28948022309329048855892746252171976963363056481941647379679742748393362948097"
```

Comment:
I believe these constants are defined somewhere else. If possible please avoid copying the same number around. This would make our lives easier.